lua/darkrp_modules/levels/sh_config.lua

```diff
@@ -19,20 +19,34 @@ LevelSystemConfiguration.GreenAllBars = true // Are the green bars at the bottom
 
 LevelSystemConfiguration.KillModule = true // Give XP + Money for kills! // Next 2 settings control this.
 LevelSystemConfiguration.Friendly = true // Only take away money / give XP if the killer is a lower level/same level than the victim. (Recommended:true)
-LevelSystemConfiguration.TakeAwayMoneyAmount = 100 // How much money to take away from players when they are killed and add to the killer. You can change this to 0 if none. The XP amount is dynamic.
+LevelSystemConfiguration.TakeAwayMoneyAmount = 0 // How much money to take away from players when they are killed and add to the killer. You can change this to 0 if none. The XP amount is dynamic.
 
 LevelSystemConfiguration.NPCXP = true // Give XP when an NPC is killed?
 LevelSystemConfiguration.NPCXPAmount = 10 // Amount of XP to give when an NPC is killed
 
 LevelSystemConfiguration.TimerModule = true // Give XP to everybody every howeverlong
-LevelSystemConfiguration.Timertime = 100 // How much time (in seconds) until everybody gets given XP
-LevelSystemConfiguration.TimerXPAmount = 100 // How much XP to give each time it goes off
-LevelSystemConfiguration.YourServerName = "on the server" // The notifcation text ish. "You got 100XP for playing on the server."
+LevelSystemConfiguration.Timertime = 120 // How much time (in seconds) until everybody gets given XP
+LevelSystemConfiguration.TimerXPAmount = 50 // How much XP to give each time it goes off
+LevelSystemConfiguration.TimerXPAmountVip = 100 // How much XP to give for vip players each time it goes off
+LevelSystemConfiguration.TimerXPVipGroups = "vip" // The vip groups
+LevelSystemConfiguration.YourServerName = "" // The notifcation text ish. "You got 100XP for playing on the server."
 
-LevelSystemConfiguration.XPMult = 1 // How hard it is to level up. 2 would require twice as much XP, ect.
+LevelSystemConfiguration.XPMult = 2 // How hard it is to level up. 2 would require twice as much XP, ect.
 LevelSystemConfiguration.MaxLevel = 99 // The max level
 LevelSystemConfiguration.ContinueXP = false // If remaining XP continues over to next levels. I recommend this to be false. Seriously. What if a level 1 gets 99999999 XP somehow? He is level 99 so quickly.
 
+//Language settings
+LevelSystemConfiguration.LangBuyEntity = 'Vous n\'avez pas le bon level pour acheter ça!' // error message when someone can't buy an entity
```

Comment:
Could you do these in English for the repo since everything else is English? I could translate for you if needed

Reply:
You could also do 
LevelSystemConfiguration.Language = {
    buyEntity = "something",
    takeJob = "whatever"
}
for clarity

Reply:
this translation isn't the defenitive, im going to make a better one in a few days